src/managers/KeybindManager.cpp

```diff
@@ -2203,7 +2203,12 @@ void CKeybindManager::moveWindowOrGroup(std::string args) {
         } else
             moveWindowOutOfGroup(PWINDOW, args);
     } else if ((*PIGNOREGROUPLOCK || !ISWINDOWGROUPLOCKED) && ISWINDOWGROUP) // no target window
+    {
```

Comment:
this brace should go before the comment on top.

Reply:
added the changes mentioned in your review